aepsych/acquisition/lookahead.py

```diff
@@ -406,12 +406,12 @@ def construct_inputs_global_lookahead(
     lookahead_type="levelset",
     target: Optional[float] = None,
     posterior_transform: Optional[PosteriorTransform] = None,
-    query_set_size: Optional[int] = 256,
+    query_set_size: int = 256,
     Xq: Optional[Tensor] = None,
     **kwargs,
-):
-    lb = [bounds[0] for bounds in kwargs["bounds"]]
-    ub = [bounds[1] for bounds in kwargs["bounds"]]
+) -> Dict[str, Any]:
+    lb = torch.tensor([bounds[0] for bounds in kwargs["bounds"]])
+    ub = torch.tensor([bounds[1] for bounds in kwargs["bounds"]])
     Xq = Xq if Xq is not None else make_scaled_sobol(lb, ub, query_set_size)
```

Comment:
We need to pass in either Xq or query_set_size. We need to add a check that errors out if both are None.

Reply:
Sure, I will add those checks after making `query_set_size` optional.